chainer/functions/loss/huber_loss.py

```diff
@@ -7,9 +7,15 @@
 
 class HuberLoss(function.Function):
 
-    def __init__(self, delta):
+    def __init__(self, delta, reduce='sum_along_second_axis'):
         self.delta = delta
 
+        if reduce not in ('sum_along_second_axis', 'no'):
+            raise ValueError(
```

Comment:
I am wondering if there is better option name than `sum_along_second_axis`. Do you have any idea?